Nodejs/Product/Nodejs/Snippets/1033/Nodejs/stringify.snippet

```diff
@@ -0,0 +1,25 @@
+﻿<?xml version="1.0" encoding="utf-8"?>
+<CodeSnippets
+    xmlns="http://schemas.microsoft.com/VisualStudio/2005/CodeSnippet">
+  <CodeSnippet Format="1.0.0">
+    <Header>
+      <Title>stringify</Title>
+      <Description>Code Snippet to stringify a Javascript value using JSON/Description</Description>
+      <Shortcut>stringify</Shortcut>
+      <SnippetTypes>
+        <SnippetType>Expansion</SnippetType>
+      </SnippetTypes>
+    </Header>
+    <Snippet>
+      <Declarations>
+        <Object>
+          <ID>object</ID>
+          <Default>object</Default>
+        </Object>
+      </Declarations>
+      <Code Language="Nodejs">
+        <![CDATA[JSON.stringify($object$);]]>

```

Comment:
JSON.stringify and JSON.parse are typically used as a part of a statement rather than the full statement itself so you probably don't want the semicolon.